apps/theviewlodge/app/root.tsx

```
@@ -88,6 +88,21 @@ export function Layout({ children }: { children: React.ReactNode }) {
                 <meta name="theme-color" content="#ede4da" />
                 <Meta />
                 <Links />
+                {/* Google tag (gtag.js) */}
+                <script
+                    async
+                    src="https://www.googletagmanager.com/gtag/js?id=AW-16672536060"
+                />
+                <script
+                    dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)